Sources/Mantis/Helpers/TransformRecord.swift

```diff
@@ -0,0 +1,85 @@
+import UIKit
+
+enum TransformType {
+    case resetTransforms
+    case transform
+}
+
+class TransformRecord: NSObject {
+    
+    let transformType : TransformType!
+    
+    var actionName : String!
+        
+    let previousValues :  Dictionary<String, Any?>!
+    let currentValues : Dictionary<String, Any?>!
```

Comment:
I see you use Dictionary<String, Any?> in a couple of places. Do we already know the exact type we will use for them? If that is the case, I suggest using the exact type. If possible, I suggest avoiding using `Any`

Reply:
changes made